README.md

```diff
@@ -2648,6 +2650,13 @@ IEEE Robotics and Automation Letters (RA-L), 2022. [Dataset](https://visibilityd
 - <a name="Gao22ral"></a>Gao, L., Liang, Y., Yang, J., Wu, S., Wang, C., Chen, J., Kneip, L.,  
 *[VECtor: A Versatile Event-Centric Benchmark for Multi-Sensor SLAM](https://doi.org/10.1109/LRA.2022.3186770)*,  
 IEEE Robotics and Automation Letters (RA-L), 7(3):8217-8224, 2022. [PDF](https://arxiv.org/abs/2207.01404), [Dataset](https://star-datasets.github.io/vector/), [MPL Calibration Toolbox](https://github.com/mgaoling/mpl_calibration_toolbox), [MPL Dataset Toolbox](https://github.com/mgaoling/mpl_dataset_toolbox).
+- <a name="Chaney23M3ED"></a>Chaney, K., Cladera, F., Wang, Z., Bisulco, A., Hsieh, M.A., Korpela, C., Kumar, V., Taylor, C.J. and Daniilidis, K.,
```

Comment:
Need a line breaks (Authors, Title and Publication venue in separate lines, please). See the preview:
![image](https://github.com/uzh-rpg/event-based_vision_resources/assets/8024432/1ca8629f-fa39-4ff5-8a40-14b53f20a34c)

- Formatting the name of the conference: see EVREAL and other CVPRW examples.
- For the name / key we typically use NameYYpublicationvenue, so it should be Chaney23cvprw
Thanks